### ISSUE TITLE
Possibility to change the core ruleset messages (as we can change severity)

### ISSUE THREAD

**Issue by arnaduga** [stoplightio/spectral#845]
**User story.**
As a API VALIDATION SCRIPT DEVELOPER, I can do replace the core rules return message (as I ca do for severity), so that I can have messages more consistent regarding a company standard guide (for instance, "Break rule 00001: information section MUST have a description".

**Is your feature request related to a problem?**
I'm always frustrated when I do not have consistent messages:
```
 2:6     error  info-description       OpenAPI object info description must be present and non-empty string.

11:10    error  https-server-endpoint  GUIDE001 - API described SHOULD be exposed on a secure http endpoint

13:11    error  verb-in-path           GUIDE003 - API paths MUST NOT contains verbs.
```

**Describe the solution you'd like**
I would like to be able to overlad rules like:
```
extends: spectral:oas
rules:
  info-description: 
    overload:
        severity: error
        message: "GUIDE0010: OpenAPI object MUST have a non-empty description field"
```

Structure of the way to overload above is just a suggestion. Distinction between string or object could be done to keep backward compatibility.

Thank you !

**Comment by P0lip** [comment]
This should be relatively easy, as the merging function already supports it.
We'd just need to update the ruleset validation function.

**Comment by P0lip** [comment]
@arnaduga here's how one can do it 
1. Create `.spectral.js` - the JS version of Spectral ruleset
2. Paste
```js
import { oas } from "@stoplight/spectral-rulesets";

const customMessages = {
  "oas3-valid-media-example": "OAS3: Examples must be valid against their defined schema.",
  "oas2-oneOf": 'OAS2: "oneOf" keyword must not be used in OpenAPI v2 document.',
  "operation-description": "Must have description.",
};

const oasRulesetWithOverriddenMessages = {
  ...oas,
  rules: Object.entries(oas.rules).reduce((rules, [name, rule]) => {
    rules[name] =
      name in customMessages
        ? {
            ...rule,
            message: customMessages[name],
          }
        : rule;

    return rules;
  }, {}),
};

export default {
  extends: [oasRulesetWithOverriddenMessages,
};
```
